### ISSUE TITLE
bug: iOS swipe to go back does not work on default tab page

### ISSUE THREAD

**Issue by andreialecu** [driftyco/ionic#4488]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>8</span>  <span ionic-webview>webview</span>

<span ionic-description>I had a pretty hard to track bug in which one of my tabs would not react in any way to the swipe from left of the screen gesture on iOS.

Swiping on other tabs worked, so while debugging I noticed that changing my `otherwise` route to a different tab made the problem occur on that tab instead. It seems to be related to the default view.

The problem can be seen in the ionic-starter-tabs project by simply changing the otherwise call to 
`$urlRouterProvider.otherwise('/tab/chats');`

Any ideas for workarounds would be highly appreciated.</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @andreialecu!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=4488). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=4488)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by andreialecu** [comment]
Note that I have only tested this in Ionic View so far, and I have noticed that this fixes it for the *ionic-starter-tabs* project: 

`<script>window.phonegap = {}</script>`

The code is from a workaround by @tlancina in https://github.com/driftyco/ionic-view-issues/issues/52#issuecomment-99250866

I stumbled upon it while also trying to debug window.cordova being undefined.

However, in my actual app, trying this workaround doesn't seem to work very well in Ionic View as I'm seeing some serious side effects. 

It seems like `$ionicPlatform.ready` takes a very long time to fire (sometimes several minutes, or never at all) and until it fires, things like the bouncing scroll and swipe to go back doesn't work at all in any of the tabs.
